packages/components/Button/src/Button.settings.win32.ts

```diff
@@ -78,9 +80,14 @@ export const settings: IComposeSettings<IButtonType> = [
         tokens: {
           backgroundColor: 'neutralBackground1Hover',
           color: 'neutralForeground1',
-          borderColor: 'strokeFocus2',
-          borderWidth: 2,
+          borderColor: 'strokeFocus2'
         },
+        stack: {
+          style: {
+            borderColor: '#000',
```

Comment:
This probably doesn't work well for a dark mode. Would it be possible for us to apply the borderColor token to the stack by adding "`{ source: 'borderColor', lookup: getPaletteFromTheme }`" to the `stack` property on of `styles` object in the `Button.tsx` file?

Reply:
it works for all the states. Can I use it only for the focused state?

Reply:
I think you can get the slotProps to be different in Button.tsx's usePrepareProps, and modify the style/borderColor of the stack depending on the current state.info.focused. It will probably take some weird/hacky code, but we're already in hacky terrirory anyway.